assets/xml/objects/object_geldb.xml

```diff
@@ -1,78 +1,84 @@
-        <Limb Name="object_geldb_Standardlimb_00A6BC" Type="Standard" EnumName="OBJECT_GELDB_LIMB_04" Offset="0xA6BC" />
-        <Limb Name="object_geldb_Standardlimb_00A6C8" Type="Standard" EnumName="OBJECT_GELDB_LIMB_05" Offset="0xA6C8" />
-        <Limb Name="object_geldb_Standardlimb_00A6D4" Type="Standard" EnumName="OBJECT_GELDB_LIMB_06" Offset="0xA6D4" />
-        <Limb Name="object_geldb_Standardlimb_00A6E0" Type="Standard" EnumName="OBJECT_GELDB_LIMB_07" Offset="0xA6E0" />
-        <Limb Name="object_geldb_Standardlimb_00A6EC" Type="Standard" EnumName="OBJECT_GELDB_LIMB_08" Offset="0xA6EC" />
-        <Limb Name="object_geldb_Standardlimb_00A6F8" Type="Standard" EnumName="OBJECT_GELDB_LIMB_09" Offset="0xA6F8" />
-        <Limb Name="object_geldb_Standardlimb_00A704" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0A" Offset="0xA704" />
-        <Limb Name="object_geldb_Standardlimb_00A710" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0B" Offset="0xA710" />
-        <Limb Name="object_geldb_Standardlimb_00A71C" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0C" Offset="0xA71C" />
-        <Limb Name="object_geldb_Standardlimb_00A728" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0D" Offset="0xA728" />
-        <Limb Name="object_geldb_Standardlimb_00A734" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0E" Offset="0xA734" />
-        <Limb Name="object_geldb_Standardlimb_00A740" Type="Standard" EnumName="OBJECT_GELDB_LIMB_0F" Offset="0xA740" />
-        <Limb Name="object_geldb_Standardlimb_00A74C" Type="Standard" EnumName="OBJECT_GELDB_LIMB_10" Offset="0xA74C" />
-        <Limb Name="object_geldb_Standardlimb_00A758" Type="Standard" EnumName="OBJECT_GELDB_LIMB_11" Offset="0xA758" />
-        <Limb Name="object_geldb_Standardlimb_00A764" Type="Standard" EnumName="OBJECT_GELDB_LIMB_12" Offset="0xA764" />
-        <Limb Name="object_geldb_Standardlimb_00A770" Type="Standard" EnumName="OBJECT_GELDB_LIMB_13" Offset="0xA770" />
-        <Limb Name="object_geldb_Standardlimb_00A77C" Type="Standard" EnumName="OBJECT_GELDB_LIMB_14" Offset="0xA77C" />
-        <Limb Name="object_geldb_Standardlimb_00A788" Type="Standard" EnumName="OBJECT_GELDB_LIMB_15" Offset="0xA788" />
-        <Limb Name="object_geldb_Standardlimb_00A794" Type="Standard" EnumName="OBJECT_GELDB_LIMB_16" Offset="0xA794" />
-        <Limb Name="object_geldb_Standardlimb_00A7A0" Type="Standard" EnumName="OBJECT_GELDB_LIMB_17" Offset="0xA7A0" />
-        <Skeleton Name="object_geldb_Skel_00A808" Type="Flex" LimbType="Standard" LimbNone="OBJECT_GELDB_LIMB_NONE" LimbMax="OBJECT_GELDB_LIMB_MAX" EnumName="ObjectGeldbLimb" Offset="0xA808" />
-        <Animation Name="object_geldb_Anim_00AA8C" Offset="0xAA8C" /> <!-- Original name is "geldB_talk" -->
-        <Animation Name="object_geldb_Anim_00B0E4" Offset="0xB0E4" /> <!-- Original name is "geldB_wait" -->
+        <Animation Name="gGerudoRedBeggingAnim" Offset="0x734" /> <!-- begging?. Original name is "ga_asonde" "idle"? -->
```

Comment:
Actually 1 other thing. Could you drop the `ing` on the anims, I don't think we've been including them. i.e. `Begging` -> `Beg`, `Running` -> `Run`, `Walking` -> `Walk`